lnst/RecipeCommon/Perf/Measurements/LatencyMeasurement.py

```diff
@@ -0,0 +1,186 @@
+import time
+import signal
+import socket
+import logging
+
+from lnst.RecipeCommon.Perf.Measurements.BaseFlowMeasurement import (
+    BaseFlowMeasurement,
+    NetworkFlowTest,
+    Flow,
+)
+from lnst.Tests.Latency import LatencyClient, LatencyServer
+
+from lnst.RecipeCommon.Perf.Measurements.Results.LatencyMeasurementResults import (
+    LatencyMeasurementResults,
+)
+from lnst.RecipeCommon.Perf.Measurements.Results.AggregatedLatencyMeasurementResults import (
+    AggregatedLatencyMeasurementResults,
+)
+
+from lnst.Controller.Job import Job
+
+from lnst.RecipeCommon.Perf.Results import (
+    PerfInterval,
+    ScalarSample,
+    ParallelPerfResult,
+    SequentialScalarResult,
+)
+from lnst.Controller.RecipeResults import ResultType
+
+
+class LatencyMeasurement(BaseFlowMeasurement):
```

Comment:
discussed on tech meeting that this could maybe work as a standard measurement which simply measures latency in a regular interval and is combined with an additional measurement which is "more primary" in the overall PerfRecipeConfiguration which has a "10 second" quiet period and then executes the poisoning, in that way you will get the following hierarchy:
```
1. cpu measurement: [....]
2. cpu measurement: [....]
3. latency measurement:   [10, 1, 1, 1, 1, 1, 100, 100, ...]
4. poisoning measurement: [0 , 0, 0, 0, 0, 100, 100, ...]
```

and afterwards you can postprocess these results to get:
```
1. cpu measurement: [....]
2. cpu measurement: [....]
3. latency measurement - start:    [10]
4. latency measurement - middle:   [1, 1, 1, 1, 1]
5. latency measurement - poisoned: [100, 100, ...]
```

and evaluate these each individually